identity/handler.go

```diff
@@ -39,6 +39,16 @@ const (
 	BatchPatchIdentitiesLimit = 2000
 )
 
+// fandom-start - add API to validate email before saving user in UCP
+const RouteValidate = RouteCollection + "/validate"
+
+// fandom-end
+
+// fandom-start allow admins to remove multifactor authentication for an identity
+const RouteMultifactor = RouteItem + "/credentials/:credentialType"
```

Comment:
Issue: API to delete 2FA is not migrated
See https://github.com/Wikia/kratos/pull/96/files#r1477324330

Reply:
I'll delete this `const` because it's unused.
The upstream implementation replaced our endpoint.
Follow-up task: https://fandom.atlassian.net/browse/PLATFORM-9038

Reply:
Unused `const` deleted.